Makefile

```diff
@@ -39,6 +39,10 @@ test-e2e:
 	@poetry run pytest $(TESTS)/$(E2E) --slow --cov --cov-config=pyproject.toml --cov-report=xml
 .PHONY: test-e2e
 
+test-int:
```

Comment:
It is a nit, but for a new-comer `int` could mean more things than `integration` which would requires them to check more to confirm. Maybe a slightly change in the name could make it more intuitive.